notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>characters -&gt; strings? (I find that a bit more common name, although in SQL land character is maybe often used)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>i'll make sure adjustments and other tutorials will be adjusted to using <code>strings</code> (cfr. https://github.com/<a href="https://github.com/pandas-dev/pandas/pull/27949" target="_blank">pandas-dev/pandas/pull/27949)</a></p>